ansible-scylla-node/tasks/RedHat.yml

```diff
@@ -99,6 +99,16 @@
       when: result.changed
   when: scylla_is_installed and not skip_installed_scylla_version_check
 
+- name: install systemd-udev
+  dnf:
+    name: systemd-udev
+  become: true
+
+- name: install policycoreutils
+  dnf:
+    name: policycoreutils
+  become: true
```

Comment:
Out of curiosity: why do you install 2 packages here each in a separate task while you do it in a single task in `check_ports_RedHat_tasks.yml` above?